WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -17,6 +17,8 @@ struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .cardPresentKnownReader:
             return buildConfig == .localDeveloper || buildConfig == .alpha
+        case .reviewOrder:
+            return buildConfig == .localDeveloper
```

Comment:
Why not .alpha too?

Reply:
Yes normally I'd add .alpha but in this case I plan to remove it as soon as I get the feature done - so I think just keeping it local is enough.